### ISSUE TITLE
🐛 Expected token ParenClose but got Word for unknown license

### ISSUE THREAD

**Issue by jellydn** [rome/tools#1202]
## Environment

```
$ rome rage --summary
Environment Variables
  Object {
    COLORFGBG: "15;0"
    COLORTERM: "truecolor"
    TERM: "xterm-256color"
    TERM_PROGRAM: "iTerm.app"
  }

User Config
  unset

Rome Version
  10.0.4-beta

Node Version
  12.18.3

Platform
  darwin x64 19.6.0

Terminal Features
  Object {
    background: "dark"
    colorDepth: 24
    columns: 98
    cursor: true
    hyperlinks: true
    isTTY: true
    unicode: true
  }

Client Flags
  Object {
    clientName: "cli"
    cwd: "/Users/huynhducdung/Projects/aaa-cards-app"
    realCwd: "/Users/huynhducdung/Projects/aaa-cards-app"
    silent: false
  }

Server Status
  Object {
    projects: Array [
      Object {id: 0}
      Object {id: 1}
    ]
    server: Object {
      heapTotal: 59_187_200
      pid: 81_761
      uptime: 0.632869425
    }
    workers: Array [
      Object {
        astCacheSize: 0
        heapTotal: 59_187_200
        ownedBytes: 0
        ownedFileCount: 0
        pid: 81_761
        uptime: 0.632793094
      }
    ]
  }


✨  Done in 1.13s.
```

## Steps To Reproduce

1. Install `dom-storage` package
2. Run `yarn rome check`
```
 node_modules/dom-storage/package.json:16:19 parse/spdxLicense ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Expected token ParenClose but got Word
```

## Expected Results
 It should work without the internal error. Probably, this is the root cause https://git.coolaj86.com/coolaj86/dom-storage.js/src/branch/master/package.json#L16

```
 "license": "(MIT or Apache-2.0)",
```

**Comment by ematipico** [comment]
Hi @jellydn, yes that's invalid license. The next release will have a configuration that would help to ignore these invalid licenses.

**Comment by ematipico** [comment]
This feature is available, we can close it

**Comment by ocotopdf** [comment]
@ematipico This is still happening when running `npx rome init` How do I change rome to a newer version?

**Comment by ocotopdf** [comment]
https://www.npmjs.com/package/rome is still pointing to the the pre-fix 10.0.4 beta release. How is this resolved for new users?

**Comment by ematipico** [comment]
The nightly release has that feature. Soon we'll publish a new latest, please bear with us a little bit longer!
